definitions/npm/sequelize_v4.x.x/flow_v0.42.x-/test_sequelize.js

```diff
@@ -247,7 +247,7 @@ Warehouse.getAssociationForAlias(WarehouseBranch, 'blah')
 product.getBarcode();
 product.getBarcode({ scope: null }).then(b => b && b.code);
 
-product.setBarcode();
+product.setBarcode(null);
```

Comment:
Not with the type def changes.  Some things like this that I got from the original typescript definitions were really weird.  I mean `setBarcode()` may have the same effect as `setBarcode(null)`, but by accident.  I can't find any examples in the sequelize docs of calling it without args, so I don't think it's the intended usage.

Reply:
And personally I would much rather everyone explicitly pass `null` or `undefined` than calling without args, so that it's obvious what the call actually does.